vm/policy.go

```diff
@@ -15,10 +15,15 @@ import (
 
 	tp "github.com/kubearmor/KubeArmor/KubeArmor/types"
 	pb "github.com/kubearmor/KubeArmor/protobuf"
+	"github.com/kubearmor/kubearmor-client/cilium"
```

Comment:
Can we directly import the typesfrom the cilium package?

https://github.com/cilium/cilium/blob/master/pkg/k8s/apis/cilium.io/v2/cnp_types.go#L32